napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -305,7 +312,11 @@ class QtCustomTitleBar(QLabel):
     """
 
     def __init__(
-        self, parent, title: str = '', vertical=False, close_btn=True
+        self,
+        parent,
+        title: str = '',
+        vertical=False,
+        close_btn=True,
```

Comment:
Why are changes in this file required? I think that it should be reverted.

Reply:
I think this was just my linter. The change isn't required, I'll revert it.